### PR TITLE
Add database tests on ALP

### PR DESCRIPTION
**Pull request by Zaoliang**
see https://progress.opensuse.org/issues/115196
https://progress.opensuse.org/issues/115196 got merged
add YAML_SCHEDULE: 'schedule/alp/alp_databases.yaml' now

### CODE REVIEW

File: job_groups/alp_poc.yaml [hunk 1]
```diff
@@ -76,6 +76,10 @@ products:
     distri: alp
     flavor: SelfInstall_NonTransactional
     version: '0.1'
+  alp-databases-x86_64:
+    distri: alp
+    flavor: SelfInstall_NonTransactional
+    version: '0.1'
```
Comment (foursixnine) [inline job_groups/alp_poc.yaml:82]: this is not needed, as we're only enabling testsuites.

**Review by foursixnine — CHANGES_REQUESTED** [review]

**Comment by foursixnine** [comment]
PR is related https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15513

File: job_groups/alp_poc.yaml (hunk 1 above)
Comment (Zaoliang) [reply job_groups/alp_poc.yaml]: updated now, thanks

**Comment by Zaoliang** [comment]
http://10.162.30.85/tests/5026# should be an example for database tests on ALP.

**Review by foursixnine — APPROVED** [review]

**Comment by foursixnine** [comment]
@Zaoliang you need to solve the conflict first :)

**Pull request closed without merging**